src/components/Answer.tsx

```diff
@@ -0,0 +1,9 @@
+import { Button } from '@chakra-ui/react';
+
+export default function Answer(props: {background: string, text: string}){
```

Comment:
Might be a good idea to assign some sort of id for answer. Would be useful when processing (i.e. sending it over to the backend)

Reply:
Good call, we'll definitely need this at some point